numpy/core/tests/test_scalarmath.py

```diff
@@ -144,8 +144,19 @@ def test_int_float_promotion_truediv(fscalar):
     # For int16, result has to be ast least float32 (takes ufunc path):
     assert (np.int16(1) / f).dtype == np.dtype("float32")
 
+def check_support_sve():
+    import subprocess
+    cmd = 'lscpu'
+    try:
+        subprocess.run(cmd, check=True)
+        process = (subprocess.Popen(cmd, stdout=subprocess.PIPE,
+                            shell=True).communicate()[0]).decode('utf-8')
+        return "sve" in process
+    except OSError:
+        return false
 
 class TestBaseMath:
+    @pytest.mark.skipif(check_support_sve, reason="gh-22982")
```

Comment:
I think I slightly prefer xfail for this one, but either is OK.
```suggestion
    @pytest.mark.xfail(check_support_sve, reason="gh-22982")
```